telegram/ext/_handlers/callbackqueryhandler.py

```diff
@@ -85,6 +85,17 @@ async def callback(update: Update, context: CallbackContext)
 
             .. versionchanged:: 13.6
                Added support for arbitrary callback data.
+        game_pattern (:obj:`str` | :func:`re.Pattern <re.compile>` | :obj:`callable` | optional)
+            Pattern to test :attr:`telegram.CallbackQuery.game_short_name` against. If a string or a
+            regex pattern is passed, :func:`re.match` is used on
+            :attr:`telegram.CallbackQuery.game_short_name` to determine if an update should be
+            handled by this handler.
+            To filter arbitrary objects you may pass:
+
+            - a callable, accepting exactly one argument, namely the
+                :attr:`telegram.CallbackQuery.game_short_name`. It must return :obj:`True` or
+                :obj:`False`/:obj:`None` to indicate, whether the update should be handled.
```

Comment:
For starters, please allow only patterns as input and not callables. In the case of `pattern`, that is mainly there for arbitrary callback data. 

That also has some implications below

Reply:
Why do you don't want callables ? 


The callables would give more flexibility. 
```python
CallbackQueryHandler(callback=cb, game_pattern=lambda _: True)
```

Reply:
patterns should cover most use cases. `game_short` name is around for a very long time and we've only recently been asked to support filtering for it in `CQH` at all. Hence, I see this as rather narrow use case and would therefore like to keep it at a minimal effort support for now :) We can add support for a callable if there are explicit requests for it. note that we never got a request for supporting callables for `pattern` either before we added arbitrary callback_data

Reply:
cool, I have removed callable-function filter from the CQH